KDIC/KdicPlayer.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  KdicPlayer.swift
+//  KDIC
+//
+//  Created by Shaun Mataire on 11/9/15.
+//  Copyright © 2015 AppDev. All rights reserved.
+//
+
+import Foundation
+import AVFoundation
+
+class KdicPlayer {
```

Comment:
KDIC should be capitalized. Not only is it the station name, it should be your class prefix, and therefore all-caps.